include/deal.II/base/parameter_handler.h

```diff
@@ -233,10 +232,11 @@ namespace Patterns
     static const double max_double_value;
 
     /**
-     * Constructor. Bounds can be specified within which a valid parameter has
-     * to be. If the upper bound is smaller than the lower bound, then the
-     * infinite interval is meant. The default values are chosen such that no
-     * bounds are enforced on parameters.
+     * Constructor. Bounds can be specified within which a valid
+     * parameter has to be. If the upper bound is smaller than the
+     * lower bound, then the entire set of real numbers is
+     * implied. The default values are chosen such that no bounds are
```

Comment:
It would be a slightly clearer if this also said 'the entire set of double precision numbers' like the class description does.